travis/build-samples.sh

```diff
@@ -18,34 +18,50 @@ function usage() {
 
 function buildProject() {
     dir=$1
+    JHIPSTER=$dir
```

Comment:
@pascalgrimaud I modified the travis scripts slightly in this commit. can you take a look?